doc/manual/castxml.1.rst

```diff
@@ -99,3 +99,52 @@ case, CastXML always adds the following predefined macros:
 
 Source files may use these to identify the tool that is actually doing the
 preprocessing even when ``--castxml-cc-<id>`` changes the predefined macros.
+
+FAQ
+===
+
+Why are C++ function bodies not dumped in XML?
+----------------------------------------------
+
+The original sponsors of the project had no need for function bodies.
+So far the authors have not had time or funding to implement the support.
+
+Is there a DTD specifying the XML format dumped?
+------------------------------------------------
+
+No.
+
+Why are most classes, structs, and enums marked as "artificial"?
+----------------------------------------------------------------
+
+The artificial attribute marks nodes that are artificially generated by the compiler
+(such as implicit copy constructors). The C++ code
+
+ | struct foo {};
+
+is almost the same as the C code
+
+ | struct foo {};
+ | typedef struct foo foo;
+
+because the C++ compiler adds the typedef artificially.
+The name foo for the struct shown above is reported as the name in the Struct
+element in the XML output, but this name was artificially generated so that
+programmers would not have to write struct foo.
```

Comment:
Actually I'm not sure this particular example (or maybe even FAQ entry) is relevant to CastXML.  I don't think we mark as much stuff artificial as gccxml did.  IIUC this was talking about plain `struct foo {};` being marked artificial.  Having the marks on implicitly generated members (e.g. default constructor) is expected and was not the purpose of this FAQ entry.